test/quantization/fx/test_quantize_fx.py

```diff
@@ -2921,6 +3035,7 @@ def forward(self, x):
 
         m = M().eval()
         m = prepare_fx(m, {"": default_qconfig})
+        print(m)
```

Comment:
nit: remove?

Reply:
yeah, but this PR is refusing to get imported, finally have it working https://github.com/pytorch/pytorch/pull/73272, will fix

Reply:
thanks